presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -1420,6 +1420,22 @@ export class QueryDetail extends React.Component {
                                             {query.queryStats.rawInputDataSize}
                                         </td>
                                     </tr>
+                                   <tr>
+                                        <td className="info-title">
+                                            Shuffled Rows
```

Comment:
I wonder if there is a way to add a "help question mark" and provides some details on this stat, e.g. explain that this is the total number of rows shuffled across all query stages.